package.json

```diff
@@ -16,14 +16,16 @@
         "prepare": "make"
     },
     "dependencies": {
+        "@wharfkit/antelope": "^1.0.5",
```

Comment:
Don't think we need this, since it's already a dependency of `@wharfkit/protocol-esr`. 

https://github.com/wharfkit/protocol-esr/blob/434cfa3df189cec7d9d28fbe445ad625637f7d8c/package.json#L21

One less thing to manage!